src/core/config/srp_client.h

```diff
@@ -257,6 +257,22 @@
 #define OPENTHREAD_CONFIG_SRP_CLIENT_UPDATE_TX_MAX_DELAY 700
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_SRP_CLIENT_UPDATE_TX_MAX_DELAY_LONG
+ *
+ * Specifies the maximum value (in msec) for the long random delay wait time before sending an update message.
+ *
+ * The random delay is chosen uniformly from the min `OPENTHREAD_CONFIG_SRP_CLIENT_UPDATE_TX_MIN_DELAY` up to max value.
+ * When there is a service change in network data (e.g., SRP unicast server change or anycast sequence number change)
+ * that requires an update, the SRP client will wait for a random delay upto long
+ * delay as specified by `kUpdateTxDelayLong` before sending an SRP
```

Comment:
Should not mention `kUpdateTxDelayLong` (as it is not defiend here in config header).